File: Learnify.Domain/Entities/Quiz.cs
using System.Text.Json.Serialization;

namespace Learnify.Domain.Entities
{
    public class Quiz : BaseEntity
    {
        public string Title { get; set; } = string.Empty;
        public Guid CourseId { get; set; }
        [JsonIgnore] // Add this attribute
        public Course? Course { get; set; }
        public virtual ICollection<Question>? Questions { get; set; }
    }
}

File: Learnify.UI/src/models/entities/Quiz.js
export class Lesson {
    constructor(
        id,
        title,
        courseId,
        course = null,
        questions = [],
    ) {
        this.id = id;
        this.title = title;
        this.courseId = courseId;
        this.course = course;
        this.questions = questions;
        
    }
}